community/about-us.md

```diff
@@ -0,0 +1,36 @@
++++
+toc=true
++++
+
+# About KOSS
+
+* We are a bunch of Open Source enthusiasts, who love to share the culture of OSS. Through events and workshops, we try to share the different aspects of Software Engineering, and onboard folks with the basic understanding and literacy of Software Development.
```

Comment:
Our main work on campus is to conduct events and workshops. That is how we mostly add value. We should include a bit more about this right?

We should include a few examples of workshops that we conduct regularly - Python Workshop, Git workshop
So that people get a clear idea of what kind of work we exactly do to promote open source.

Reply:
Let me link the events page and add a section - 
https://github.com/kossiitkgp/docs/tree/master/events

Reply:
Done in line number 9